skops/card/tests/test_card.py

```diff
@@ -26,6 +26,29 @@ def fit_model():
     return reg
 
 
+def save_model_to_file(model_instance, suffix):
+    save_file = tempfile.mkstemp(suffix=suffix, prefix="skops-test")[1]
+    if suffix in (".pkl", ".pickle"):
+        with open(save_file, "wb") as f:
+            pickle.dump(model_instance, f)
+    elif suffix == ".skops":
+        dump(model_instance, save_file)
+    return save_file
+
+
+@pytest.mark.parametrize("suffix", [".pkl", ".pickle", ".skops"])
+def test_load_model(suffix):
+    model0 = LinearRegression(n_jobs=123)
+    save_file = save_model_to_file(model0, suffix)
+    loaded_model_str = _load_model(save_file)
+    save_file_path = Path(save_file)
+    loaded_model_path = _load_model(save_file_path)
+
+    assert loaded_model_str.n_jobs is model0.n_jobs
+    assert loaded_model_path.n_jobs is model0.n_jobs
+    assert loaded_model_path.n_jobs is loaded_model_str.n_jobs
```

Comment:
```suggestion
    assert loaded_model_str.n_jobs == model0.n_jobs
    assert loaded_model_path.n_jobs == model0.n_jobs
    assert loaded_model_path.n_jobs == loaded_model_str.n_jobs
```

Although `is` works in this case, let's use `==` to compare numbers and then we don't have to worry about the size of the number or the Python interpreter :)